e2e/playwright/point-click.spec.ts

```diff
@@ -1182,9 +1182,11 @@ shellSketchOnFacesCases.forEach((initialCode, index) => {
         commandName: 'Shell',
       })
       await clickOnCap()
+      await page.waitForTimeout(500)
       await cmdBar.progressCmdBar()
       await page.waitForTimeout(500)
       await cmdBar.progressCmdBar()
+      await page.waitForTimeout(500)
```

Comment:
More a workaround than a fix. But these two definitely help reducing the flakes locally on my ubuntu machine.